cmds/ocm/commands/ocmcmds/common/options/optutils/registration.go

```diff
@@ -34,7 +44,7 @@ type RegistrationOption struct {
 	Registrations []*Registration
 }
 
-const RegistrationFormat = "<name>[:<artifact type>[:<media type>]]=<JSON target config"
+const RegistrationFormat = "<name>[:<artifact type>[:<media type>[:<prio>]]]=<JSON target config>"
```

Comment:
Since it's user facing, can we please use complete words instead of abbreviations?

```suggestion
const RegistrationFormat = "<name>[:<artifact type>[:<media type>[:<priority>]]]=<JSON target config>"
```